docs/contributing/development-guide.md

```diff
@@ -0,0 +1,345 @@
+stravalib/tests/functional/__init__.py                      20     20     0%
+stravalib/tests/functional/test_client.py                  235    235     0%
+stravalib/tests/functional/test_client_rate_limiter.py      32     32     0%
+stravalib/tests/functional/test_client_write.py             40     40     0%
+stravalib/tests/functional/test_result_iterator.py          33     33     0%
+stravalib/tests/integration/__init__.py                      0      0   100%
+stravalib/tests/integration/conftest.py                     11      0   100%
+stravalib/tests/integration/strava_api_stub.py              67     13    81%
+stravalib/tests/integration/test_client.py                 124      0   100%
+stravalib/tests/unit/__init__.py                             0      0   100%
+stravalib/tests/unit/test_attributes.py                     63      0   100%
+stravalib/tests/unit/test_client_utils.py                   35      0   100%
+stravalib/tests/unit/test_limiter.py                        75      0   100%
+stravalib/tests/unit/test_model.py                          64      0   100%
+stravalib/unithelper.py                                     16      1    94%
+stravalib/util/__init__.py                                   0      0   100%
+stravalib/util/limiter.py                                  122     27    78%
+----------------------------------------------------------------------------
+TOTAL   
+```
+
+### Code coverage reporting on pull requests with codecov 
+We use an integration with [codecov.io](https://about.codecov.io) to report test coverage 
+changes on every pull request. This report will appear in your pull request once 
+all of the GitHub action checks have run. 
+
+```{note}
+The actual code coverage report is 
+uploaded on the GitHub action run on `ubuntu` and `Python 3.9`. When that step in the 
+actions completes, the report will be processed and returned to the pull request. 
+```
+
+## Documentation
+`Stravalib` documentation is created using `sphinx` and the 
+[`furo`](https://pradyunsg.me/furo/quickstart/) theme.
+`Stravalib` documentation is hosted on [ReadtheDocs](https://readthedocs.org).
+
+The final online build that you see on readthedocs happens on the readthedocs 
+website. Our continuous integration GitHub action only tests that the documentation 
+builds correctly. It also tests for broken links. 
+
+The readthedocs build is configured using the `.readthedocs.yml` file rather than 
+from within the readthedocs interface as recommended by the readthedocs website. 
+
+The badge below (also on our `README.md` file) tells you whether the 
+readthedocs build is passing or failing.
+
+[![Documentation Status](https://readthedocs.org/projects/stravalib/badge/?version=latest)](https://stravalib.readthedocs.io/en/latest/?badge=latest)
+
+Currently @hozn and @lwasser have access to the docs build 
+**#TODO: Add jonatan? just so someone else can troubleshoot?**
```

Comment:
ok just in case it's nice for someone else to have access :) do you have a readthedocs account? if not [can you please signup here](https://readthedocs.org/accounts/signup/) you can use your github account. That give me your username and i'll add you. Fixes will be  rare and i can  handle it i just worry if i'm offline and something comes up it's nice to have others with access. hans has access too.

Reply:
My username at readthedocs is `jsamoocha` (but not linked to my Github account (yet)).